Makefile

```diff
@@ -125,8 +130,6 @@ distclean:
 
 check: test
 
-# Remove files generated by running coverage, put the non-instrumented lib back
-# in place
```

Comment:
should this be removed or just moved down to the next line?

Reply:
Yes..why are we removing this?

Reply:
My bad, I made it an inline comment, then decided the help was getting too long and removed it again. Fixed.